src/Sylius/Bundle/AdminBundle/Resources/views/Customer/Show/Details/_email.html.twig

```diff
@@ -11,8 +11,8 @@
             <i class="{{ user.verified ? 'green checkmark' : 'red remove' }} icon"></i>
             {{ 'sylius.ui.email_verified'|trans }}
         </div>
+        {% if is_shop_enabled() and user.locked != 1 %}
```

Comment:
I have thought about this logic again. You could also make sure that `is_shop_enabled()` is not `true`. As an alternative to `is_shop_enabled()` you could also check it with `user.enabled`:

```twig
{% if user.enabled and user.locked != 1 %}
```

Reply:
I also suspect that `shop_is_enabled()` is currently always false and therefore the check has not worked so far. As I understand it, this function should actually check whether the customer user is enabled for the store, and then enable or disable the Imporsonate Button accordingly.

Reply:
The `shop_is_enabled()` function only checks whether ShopBundle is loaded and enabled since you can use AdminBundle independently.